Formula/goto.rb

```diff
@@ -0,0 +1,17 @@
+class Goto < Formula
+  desc "Shell tool for navigation to aliased directories with auto-completion"
+  homepage "https://github.com/iridakos/goto"
+  url "https://github.com/iridakos/goto/archive/v1.2.2.tar.gz"
+  sha256 "611ef7b82f6798ef83db0473811740c7f1ff5a15bb4ff52a1e5841bac87ba7f5"
+
+  bottle :unneeded
+
+  def install
+    bash_completion.install "goto.sh" => "goto"
```

Comment:
Is the goto script specific to bash?

Reply:
Yes, at the moment it has only been tested in Bash under macOS.

Reply:
Then let's edit the description to be `Bash tool` instead of `Shell tool`

Reply:
Sure thing, brewing fix.

Reply:
Question: there has been a new release. May I change the url as well pointing it to it? If yes, I will also update this thread to "goto 1.2.2 (new formula)"

Reply:
Yes, update everything. 1 commit.

Reply:
> Yes, update everything. 1 commit.

Sorry, I already sent the previous fixes explicitly for each review note...

Reply:
Done.